HW3_Star.java

```diff
@@ -0,0 +1,71 @@
+public class HW3_Star {
+    //    Написати методи, які виводять на екран такі “ялиночки”
+    public static void Star1 () {
+
+        final int length = 6;
+
+        for (int i = 1; i < length; i++) {
+
+            //Print spaces first
+            for (int j = length - 1; j > i; j--) {
+                System.out.print(" ");
+            }
+
+            //Then print "*"
+            for (int j = 1; j <= i; j++) {
+                System.out.print("*");
+            }
+
+            System.out.println();
+        }
+    }
+
+    public static void Star2 () {
+
+        final int length = 6;
+
+        for (int i = 6; i < length; i++) {
+
+            //Print spaces first
+            for (int j = length - 6; j > i; j--) {
+                System.out.print(" ");
+            }
+
+            //Then print "*"
+            for (int j = 6; j <= i; j++) {
+                System.out.print("*");
+            }
+
+            System.out.println();
+        }
+    }
+    public static void Star3 () {
+
+        final int length = 6;
+
+        for (int i = 6; i < length; i++) {
+
+            //Print spaces first
+            for (int j = length - 6; j > i; j--) {
+                System.out.print(" ");
+            }
+
+            //Then print "*"
+            for (int j = 6; j <= i; j++) {
+                System.out.print("*");
+            }
+
+            for (int j = length - 6; j > i; j--) {
+                System.out.print(" ");
+            }
+
+            System.out.println();
+        }
+    }
+    public static void main(String[] args) {
+        Star1();
```

Comment:
Only 1 tree is shown
![dsvds](https://github.com/TanyaChas/Les_1/assets/70166140/8a8534d4-1733-43aa-88fc-e558869a0fd9)

Reply:
public static void printFirstChristmasTree() {
        int i;
        int j;
        for(i = 1; i <= 5; ++i) {
            for(j = 1; j <= i; ++j) {
                System.out.print("x");
            }

            System.out.println();
        }

        for(i = 4; i >= 1; --i) {
            for(j = 1; j <= i; ++j) {
                System.out.print("x");
            }

            System.out.println();
        }

    }

    public static void printSecondChristmasTree() {
        for(int i = 4; i >= 1; --i) {
            for(int j = 1; j <= i; ++j) {
                System.out.print("x");
            }

            System.out.println();
        }

    }

    public static void printThirdChristmasTree() {
        for(int i = 1; i <= 5; ++i) {
            for(int j = 1; j <= i; ++j) {
                System.out.print("x");
            }

            System.out.println();
        }

        System.out.println();
    }